wiki/Storyboard/Scripting/Cheat_Sheet/en.md

```diff
@@ -159,7 +159,7 @@ becomes
 
 `_M,0,1000,,320,240,320,240`
 
-Please note the blank space in the endtime field - this is \[b\]required\[/b\].
+Please note the blank space in the endtime field—this is \[b\]required\[/b\].
```

Comment:
Should this be
```suggestion
Please note the blank space in the endtime field—this is **required**.
```
or is this part of the described syntax?

Reply:
can do, since we're doing other miscellaneous fixes along the way too

Reply:
Removed the BBCode tags in this PR because they seemed to be present only in this file.